syntax = "proto3";

import "Common.proto";

package proto;

message S2C_EnterRoom
{
    int64   id = 1;
    bool    success = 2;
}

message S2C_Chat
{
    int64   id = 1;
    string  message = 2;
}
